### PR TITLE
#8: Setup script improvements.

### PR DESCRIPTION
**Pull request by boonya**

### CODE REVIEW

**Comment by boonya** [comment]
If you want to check how it works, you can use this command: `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ets-labs/vimrc/8-improve-setup-script/setup.sh)"`

**Comment by boonya** [comment]
BTW, I have checked it on Mac OSX and Ubuntu 14.04.3 LTS.
Works for both.

**Comment by rmk135** [comment]
Absolutely gorgeous! Thanks!

File: README.rst [hunk 1]
```diff
@@ -13,18 +13,11 @@ Requirements
 Installation guide
 ------------------
 
+You can install it by using CLI just have next command executed
```
Comment (rmk135) [inline README.rst:16]: I've just realized that there should be one `\n` before / after rst `..code-block::`'s. Currently our markup has been broken a bit... Can you please fix it?

**Pull request merged**